### PR TITLE
Issue 43661: Excessive logging when indexing a .log file containing "\u" that confuses TabLoader

### PR DESCRIPTION
**Pull request by labkey-jeckels**
#### Rationale
Search indexing is treating pipeline job log files, where each line's timestamp contains a comma, as a CSV. That means we treat "\u" as a Unicode escape sequence and log for every instance where it's not the expected Unicode syntax, which can quickly overwhelm the log

#### Changes
* When indexing, don't treat \u as a Unicode escape for sniffed CSV files

### CODE REVIEW

**Comment by labkey-jeckels** [comment]
@labkey-adam After I investigated how a CSV-style TabLoader is getting chosen here it seemed safest to disable the Unicode decoding altogether. I could have gone with the message suppression after the first N lines per file instead. Let me know if you'd prefer that approach.

File: api/src/org/labkey/api/reader/TabLoader.java [hunk 1]
```diff
@@ -107,7 +107,7 @@ public DataLoader createLoader(File file, boolean hasColumnHeaders, Container mv
 
         @NotNull @Override
         // A DataLoader created with this constructor does NOT close the reader
```
Comment (labkey-adam) [inline api/src/org/labkey/api/reader/TabLoader.java]:
```suggestion
        // A TabLoader created with this constructor does NOT close the reader
```

File: api/src/org/labkey/api/reader/TabLoader.java [hunk 2]
```diff
@@ -121,21 +121,26 @@ public DataLoader createLoader(InputStream is, boolean hasColumnHeaders, Contain
...
+        private TabLoader configParsing(TabLoader loader)
+        {
             loader.setInferTypes(false);
+            // Issue 43661 - Excessive logging when indexing a .log file containing backslash followed by "u" that confuses TabLoader
+            loader.setUnescapeBackslashes(false);
             return loader;
         }
 
         @NotNull @Override
         // A DataLoader created with this constructor does NOT close the reader
```
Comment (labkey-adam) [inline api/src/org/labkey/api/reader/TabLoader.java]:
```suggestion
        // A TabLoader created with this constructor does NOT close the reader
```

**Review by labkey-adam — APPROVED** [review]
Couple comment update suggestions

**Pull request merged**